pyeudiw/federation/jwtse.py

```diff
@@ -0,0 +1,139 @@
+import base64
```

Comment:
This file must be removed and the current modules jwt and jwk must then be used

requirement: add in the backend settings the unallowed sign and enc algs (eg: RSA_1_5)